packages/frontend/scss/dialogs/_group-styles.scss

```diff
@@ -72,3 +72,13 @@ input.group-name-input {
     }
   }
 }
+
+.group-related-chats-list-wrapper {
+  margin-left: 20px;
+  margin-right: 10px;
+}
+
+.group-member-contact-list-wrapper {
+  margin-left: 10px;
+  margin-right: 5px;
```

Comment:
I think it would be nice to add comments about how these numbers are picked, and that they are also linked with the ones in the ViewProfile dialog.

Reply:
I think these numbers are a result that there was never a real desing concept behind this dialog, so that's why it had different indentations for the dialog header, for the separator and for the items. (and before this and the other PR even for different item categories) But since the lists are also used in other places I didn't want to adapt the list items. That's why we have this inconsistent mix of margins to get a consistent result.